golang/1.13/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM golang:1.13-alpine3.11
+
+RUN apk add git make --update --no-cache \
+    && go get -u golang.org/x/lint/golint \
+    mkdir /app && chmod 777 /app
```

Comment:
Please format this in a consistent manner, e.g. '&&' to end of the line along with a single command.